packages/ember-glimmer/lib/helpers/inline-if.js

```diff
@@ -0,0 +1,11 @@
+import { toBool as emberToBool } from './if-unless';
+
+//TODO: GJ: docs
```

Comment:
@rwjblue says the [docs for if](https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/helpers/if_unless.js#L9) should be moved to this file.